cypress/support/api/fees-fines.js

```diff
@@ -37,3 +37,11 @@ Cypress.Commands.add('deleteFeesFinesTypeApi', (feesFinesTypeId) => {
       path: `feefines/${feesFinesTypeId}`,
     });
 });
+
+Cypress.Commands.add('getUserFeesFines', (userId, status = 'open') => {
```

Comment:
Move it to fragments

Reply:
the structure of the DOM does not allow us to do this. in the future, when the developers fix these elements, I will change this interactor

Reply:
done